### PR TITLE
fix(ppr-demos): unify DTDL model

### PR DESCRIPTION
**Pull request by ridomin**
# Checklist
- [X ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-node/blob/master/.github/CONTRIBUTING.md).
- [] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [] I edited the corresponding document in the `devdoc` folder and added or modified requirements.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

 The DigitalTwins/PnP samples are not consistent with samples in other SDKs. 

# Description of the solution

This PR includes the same DTDL model as the C-SDK samples, and updates the component names, and telemetry names.

### CODE REVIEW

**Comment by YoDaMa** [comment]
> /cc @YoDaMa @jebrando @zhvolosh @elhorton @ericmitt

please see most recent merged change....

File: digitaltwins/samples/device/javascript/environmentalinterface.js [hunk 1]
```diff
@@ -9,14 +9,14 @@ const Command = require('azure-iot-digitaltwins-device').Command;
 
 module.exports.EnvironmentalSensor = class EnvironmentalSensor extends BaseInterface {
   constructor(name, propertyCallback, commandCallback) {
-    super(name, 'dtmi:contoso:com:EnvironmentalSensor;1', propertyCallback, commandCallback);
+    super(name, 'dtmi:YOUR_COMPANY_NAME_HERE:EnvironmentalSensor;1', propertyCallback, commandCallback);
     this.temp = new Telemetry();
     this.humid = new Telemetry();
     this.state = new Property();
     this.blink = new Command();
-    this.turnOff = new Command();
```
Comment (YoDaMa) [inline digitaltwins/samples/device/javascript/environmentalinterface.js:17]: turnoff is not one word so i don't see how this follows the proper node.js camelcase naming conventions? Are we enforcing that all properties must be lowercase?

File: digitaltwins/samples/device/javascript/sample_device.js [hunk 2]
```diff
@@ -28,10 +28,10 @@ const commandHandler = (request, response) => {
 };
 
 
-const environmentalSensor = new EnvironmentalSensor('environmentalSensor', propertyUpdateHandler, commandHandler);
+const environmentalSensor = new EnvironmentalSensor('sensor', propertyUpdateHandler, commandHandler);
```
Comment (YoDaMa) [inline digitaltwins/samples/device/javascript/sample_device.js:31]: sensor seems vague. Why is it just sensor?

File: digitaltwins/samples/device/javascript/sample_device.js [hunk 3]
```diff
@@ -28,10 +28,10 @@ const commandHandler = (request, response) => {
...
 const deviceInformation = new DeviceInformation('deviceInformation');
 
-const capabilityModel = 'dtmi:azureiot:samplemodel;1';
+const capabilityModel = 'dtmi:YOUR_COMPANY_NAME_HERE:sample_device;1';
```
Comment (YoDaMa) [inline digitaltwins/samples/device/javascript/sample_device.js:34]: @anthonyvercolano and I prefer the updated `contoso_device_corp` to `YOUR_COMPANY_NAME_HERE`

File: digitaltwins/samples/device/javascript/environmentalinterface.js (hunk 1 above)
Comment (rido-min) [reply digitaltwins/samples/device/javascript/environmentalinterface.js:17]: This should be consistent with the DTDL  interface where the command name is defined as `turnon` lowercase.

File: digitaltwins/samples/device/javascript/sample_device.js (hunk 2 above)
Comment (rido-min) [reply digitaltwins/samples/device/javascript/sample_device.js:31]: ... from the DTDL model

File: digitaltwins/samples/device/javascript/sample_device.js (hunk 3 above)
Comment (rido-min) [reply digitaltwins/samples/device/javascript/sample_device.js:34]: Can you sync with the C-SDK samples? I'm just trying to be consistent with the samples

**Comment by YoDaMa** [comment]
ok I'm going to close this PR since it is now out of date.

File: digitaltwins/samples/device/javascript/environmentalinterface.js (hunk 1 above)
Comment (ericmitt) [reply digitaltwins/samples/device/javascript/environmentalinterface.js:17]: It is PnP who impose this case. not jscript naming rule here!

File: digitaltwins/samples/device/javascript/sample_device.js (hunk 3 above)
Comment (ericmitt) [reply digitaltwins/samples/device/javascript/sample_device.js:34]: +1

**Pull request closed without merging**